spring-cloud/spring-cloud-functions/spring-cloud-function-aws/src/main/java/com/baeldung/spring/cloudfunction/CloudFunctionAwsApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.spring.cloudfunction;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+@SpringBootApplication
+public class CloudFunctionAwsApplication {
```

Comment:
The duplication bugs me a bit.

One thing to remember that is different about writing code in a tutorial setting is that we may have many more tutorials on Spring Cloud Function, each with their own setup. 

Is it possible to have the web and aws dependencies in the same pom and then make sure that your package naming convention leaves room for other demos in the future to use your same module, but isolate themselves to their own package? This tends to be a bit more ideal long-term.